DOCUMENTATION/content/getting-started/index.md

```diff
@@ -221,8 +221,12 @@ Or you can change them and rebuild the container.
 <br>
 5 - Open your browser and visit your localhost address. 
 
+Make sure you add use the right port number as provided by your running server.
+
+[http://localhost:80](http://localhost:80)
```

Comment:
As 80 is a default port, we don't need to have suffix :80.
Would you please help to update it?

Reply:
Dig it! Fixed.